dapp/src/utils/web3.js

```diff
@@ -1,9 +1,5 @@
 const networkInfo = {
   1: 'Mainnet',
-  3: 'Ropsten',
```

Comment:
I think we can keep this in. From what I remember these map nework IDS to names. Could still be useful for the dapp to be able to tell the name of the network when a user switches to a network the dapp does not support.

Reply:
Have checked the code, currently we fetch the name of the network only when successfully connected. So for current functionality of the dapp it is ok that they stay removed.